index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+  integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+  crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+  <title>Document</title>
+</head>
+<body>
+  <div class="container">
+  <header>
+  <h3>My Logo</h3>
+  <i class="fa-solid fa-bars fa-2x icon"></i>
+  </header>
+  <section class="contain">
+    <h1>
+      I’m Yacos
+      Glad to see you!
+    </h1>
+    <p>
+      I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+      experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+    </p>
+    <h5>
+      LET’S CONNECT
+    </h5>
+    <ul>
+      <li><i class="fa-brands fa-twitter socialMedia"></i></li>
+      <li><i class="fa-brands fa-linkedin-in socialMedia"></i></li>
+      <li><i class="fa-solid fa-m socialMedia"></i></li>
+      <li><i class="fa-brands fa-github socialMedia"></i></li>
+      <li><i class="fa-regular fa-hand-peace socialMedia"></i></li>
+    </ul>
+  </section>
```

Comment:
- [ ] As stated in the project requirement, Kindly I recommend you to use  `flexbox` In order to lay out the elements on the page for headline section.

Reply:
The content are already align in the right way so should i use flexbox anyway? @Abenezer-Tilahun